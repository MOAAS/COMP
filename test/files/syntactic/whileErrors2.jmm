import static io.println(int); 


class test{ 
    int i; 
    public int func(boolean a) {
        while(a+;) {
            
        }
        while(2 + 2 < 8) {
            x = x + 1;
        }
        while(bro+;ken/;) {
            x = x - 1;
        }

        while((err+;o)) {
            a = 2+2;
            while(outroerr;o) {
                while(ola+) {
                    c = c+1;
                }
            }
            c = c+3;
        }

        while((if)) {
            a = 1;
        }
        while(((((((2))))))) a=3;
        while(((((((((((123 */ 2;)))))))))))
        while(1231312 + &&;) {
            while(a+;) a=1;
        }
        while((while)) {
        
        }
        while(else;) i=1;
    }
}