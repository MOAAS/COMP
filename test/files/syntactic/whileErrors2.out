Parse Error found on while expression: line 7, column 17.
Error Details:
	The expression value is invalid! An ";" is not valid in this context.

Parse Error found on while expression: line 13, column 19.
Error Details:
	The expression value is invalid! An ";" is not valid in this context.

Parse Error found on while expression: line 17, column 20.
Error Details:
	The expression value is invalid! An ";" is not valid in this context.

Parse Error found on while expression: line 19, column 27.
Error Details:
	The expression value is invalid! An ";" is not valid in this context.

Parse Error found on while expression: line 20, column 27.
Error Details:
	The expression is missing an argument after the operator "+"

Parse Error found on while expression: line 27, column 16.
Error Details:
	The expression value is invalid! An "if" is not valid in this context.

Parse Error found on while expression: line 31, column 30.
Error Details:
	The expression is missing an argument after the operator "*"

Parse Error found on while expression: line 32, column 25.
Error Details:
	The expression is missing an argument after the operator "+"

Parse Error found on while expression: line 33, column 21.
Error Details:
	The expression value is invalid! An ";" is not valid in this context.

Parse Error found on while expression: line 35, column 16.
Error Details:
	The expression value is invalid! An "while" is not valid in this context.

Parse Error found on while expression: line 38, column 15.
Error Details:
	The expression value is invalid! An "else" is not valid in this context.

